common/src/main/resources/cache.properties

```diff
@@ -0,0 +1,3 @@
+tds.cache.expire.time.short=40
```

Comment:
I don't think this needs to be in common. You already define the defaults in the class. Individual service projects can contain this file or pass in arguments.